### PR TITLE
Update Return Values

### PR DESCRIPTION
**Pull request by mikhailnathanielabordo**
## Description
- Change the Return Value to match the `http://api.ipstack.com/` format.
- Here is a sample return we would like to replicate `http://api.ipstack.com/54.204.30.246?access_key=d7596977461fc2fdde750c2c5b982278`
- Since the API we are using to get the GeoLocation has a limit for `10,000 request/month` we decided to host this on our own.

### CODE REVIEW

**Comment by jann-Briza** [comment]
Wow what is this project 😄

File: api.py [hunk 1]
```diff
@@ -11,7 +11,51 @@ def __init__(self, reader):
...
+                result['country_code'] = info['country']['iso_code']
+                result['country_name'] = info['country']['names']['en']
+
+            if 'location' in info:
+                result['latitude'] = info['location']['latitude']
+                result['longitude'] = info['location']['longitude']
+                result['location']['time_zone'] = info['location']['time_zone']
+
+            return result
         except ValueError:
```
Comment (ninotosh) [inline api.py]: This should be right after `self.reader.get(ipv4)` because `self.reader.get(ipv4)` is the only line that raises a `ValueError`, I guess.

**Review by ninotosh — COMMENTED** [review]
Please be consistent between `'` and `"`. We usually use `'`.

File: api.py (hunk 1 above)
Comment (DarylPenetrante) [reply api.py]: I agree. we limit the try clause to the absolute minimum amount of code necessary to avoid masking bugs.

**Review by ninotosh — APPROVED** [review]
LGTM
BTW, how is the location info used?
There is no Pivotal story described, I have no context at all :sweat_smile:

**Comment by mikhailnathanielabordo** [comment]
> LGTM
> BTW, how is the location info used?
> There is no Pivotal story described, I have no context at all 😅

I just display all the API response and try to copy the structure of this `http://api.ipstack.com/54.204.30.246?access_key=d7596977461fc2fdde750c2c5b982278`

**Comment by ninotosh** [comment]
> I just display all the API response and try to copy the structure of this `http://api.ipstack.com/54.204.30.246?access_key=d7596977461fc2fdde750c2c5b982278`

Where do you display it?

**Review by eshleebien — APPROVED** [review]

**Pull request merged**